### DISCUSSION TITLE
扣666送5000纠缠

### DISCUSSION THREAD

**Discussion by shen-sheng-wq** [share121/inter-knot#1899] [其他]
本来我也不信的，结果随手试了一下，结果真的到账了。
是的粉丝宝宝们，这是米忽悠给我们玩家的龙王补偿，只要在本帖下方回复666即可到账

**Comment by CloudSwordSage** [comment]
999 (我要 1000 纠缠！！！)

**Comment by ZhuYuan-148** [comment]
各位市民，注意不要掉进虚假诈骗的陷阱---治安局反诈骗

**Reply by shen-sheng-wq** [reply]
我们这个活动不真但假

**Comment by zhima0807** [comment]
666666666666666666666666666666666666666666666

**Comment by nayina1145141919810** [comment]
10086

**Comment by csdylhy** [comment]
666666666

**Comment by capacitor1** [comment]
666送5000纠缠

**Comment by lllliufeng** [comment]
不如来█网
[http://www.绳网.top](https://www.bilibili.com/video/BV1GJ411x7h7/)

**Reply by HFOxin** [reply]
是的，这个网站可好用了，可能会成为第二个绳网

**Reply by 00zzxx** [reply]
**我就知道**

**Reply by Nie7bai** [reply]
不是，哥们

**Reply by Nie7bai** [reply]
哥们把你揣兜里，你把哥们踹沟里是吧

**Reply by I-ikun** [reply]
每天上一当，当当不重样

**Comment by Cyu7M** [comment]
66666666666666

**Comment by 1451374815** [comment]
999

**Comment by yileinafs** [comment]
6666666666666666666666666

**Comment by xixi236593** [comment]
6666666666666666666666

**Comment by xixi236593** [comment]
6666666666666

**Comment by The-Astley** [comment]
就我不知道纠缠是什么吗？（ip：新艾利都）

**Comment by bai-shengjiang** [comment]
666？666。666！

**Comment by YING-ADX** [comment]
我是秦始皇

**Reply by qiang-di** [reply]
我信你

**Comment by qiang-di** [comment]
666

**Comment by Cherss111** [comment]
我信你，666

**Comment by HAZDSB** [comment]
666

**Comment by light814** [comment]
666大哥真送嘛⊙ω⊙

**Comment by RIAOBTNO** [comment]
666

**Comment by wyt080125** [comment]
666

**Comment by littlehorseaaa** [comment]
66666

**Comment by yingguan-Cloud** [comment]
666<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 1luik** [comment]
666

**Comment by shxtxdy** [comment]
666

**Comment by Landseasky** [comment]
1145141919810

**Comment by WFGoose** [comment]
事已至此，还是先联系朱鸢警官吧#-_-||

**Reply by Rolesican** [reply]
不是，哥们

**Comment by LIU2-GIT** [comment]
666

**Comment by fssasfasd** [comment]
666真送吗哥<img src="https://img-static.mihoyo.com/communityweb/upload/523a91a1b8b421fd065e798e63a4fc3a.png" alt="YUI-星星眼" width="75" >

**Comment by DaAiLangZun** [comment]
666

**Comment by tohka-722** [comment]
6666666

**Comment by Pheathon** [comment]
666666666666

**Comment by 3283276594** [comment]
666
